Sources/SmileID/Classes/Metadata/BundleExtension.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+extension Bundle {
+    /// Gets the host application name and version in the format "App Name v1.0.0"
+    var hostApplicationInfo: String {
+        // Get the app name (display name or bundle name)
+        let appName = self.object(forInfoDictionaryKey: "CFBundleDisplayName") as? String 
+            ?? self.object(forInfoDictionaryKey: "CFBundleName") as? String 
+            ?? "Unknown App"
+
+        // Get the app version
+        let appVersion = self.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String 
+            ?? "Unknown Version"
+
+        // Return in format "App Name v1.0.0"
+        return "\(appName) v\(appVersion)"
```

Comment:
@tobitech can we clean this up so that we return the following:
 * if appName, appVersion -> "\(appName) v\(appVersion)"
 * if appName, !appVersion -> "\(appName)"
 * otherwise -> "unknown"
See https://github.com/smileidentity/android/blob/1f2805b009d2868dd5cb90bcb891284bd5846c74/lib/src/main/java/com/smileidentity/models/v2/metadata/ApplicationInfoProvider.kt#L9 for example